cmd/podman/volumes/export.go

```diff
@@ -0,0 +1,82 @@
+package volumes
+
+import (
+	"context"
+
+	"github.com/containers/podman/v3/cmd/podman/common"
+	"github.com/containers/podman/v3/cmd/podman/inspect"
+	"github.com/containers/podman/v3/cmd/podman/registry"
+	"github.com/containers/podman/v3/libpod/define"
+	"github.com/containers/podman/v3/pkg/domain/entities"
+	"github.com/containers/podman/v3/utils"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var (
+	volumeExportDescription = `
+podman volume export
+
+Allow content of volume to be exported into external tar.`
+	exportCommand = &cobra.Command{
+		Use:               "export [options]",
```

Comment:
This line is wrong
```suggestion
		Use:               "[options] VOLUME",
```

Reply:
@Luap99 this is needed for correct formatting of `podman volume --help`.

Reply:
Sorry, yes `export` has to stay my bad.

Reply:
The VOLUME part is missing, this is causing the completion test to fail.